worlds/sm/variaRandomizer/utils/parameters.py

```diff
@@ -61,7 +61,7 @@ def diff4solver(difficulty):
         return "mania"
 
 # allow multiple local repo
-appDir = Path(__file__).parents[4]
+appDir = os.path.join(os.path.dirname(sys.executable), "lib") if getattr(sys, "frozen", False) else Path(__file__).parents[4]
```

Comment:
`os.path.join(os.path.dirname(sys.executable), "lib") if getattr(sys, "frozen", False)`
Is a bad assumption that only works with our current configuration/build process and is not resilient to any changes.